crates/turbopack-ecmascript-runtime/js/src/dev/runtime/base/runtime-base.ts

```diff
@@ -15,7 +15,7 @@
 
 // This file must not use `import` and `export` statements. Otherwise, it
 // becomes impossible to augment interfaces declared in `<reference>`d files
-// (e.g. `Module`). Hence the need for `import()` here.
+// (e.g. `Module`). Hence, the need for `import()` here.
```

Comment:
my IDE made a bunch of grammar suggestions 🙃

Reply:
TIL!